src/Tools/ExternalAccess/AspNetCore/Internal/EmbeddedLanguages/AspNetCoreRouteEmbeddedLanguageClassifier.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Composition;
+using Microsoft.CodeAnalysis.Classification;
+using Microsoft.CodeAnalysis.ExternalAccess.AspNetCore.EmbeddedLanguages;
+using Microsoft.CodeAnalysis.Host.Mef;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.AspNetCore.Internal.EmbeddedLanguages
+{
+    [ExportEmbeddedLanguageClassifierInternal(
+        nameof(AspNetCoreRouteEmbeddedLanguageClassifier), LanguageNames.CSharp, supportsUnannotatedAPIs: false, "Route"), Shared]
```

Comment:
from an RPS perspective, we should be ok here.  We use these constants so taht our completion engine does not try to instantiate these providers unless it sees a StringSyntaxAttribute with "Route" as the argument.  So in RPS tests we should not load anything in this dll.

(this is similar to how we make sure that in C# scenarios we're not loading VB features).